.flake8

```diff
@@ -1,2 +1,3 @@
 [flake8]
 exclude = .git,__pycache__,build,dist,versioneer.py,atef/_version.py,docs/source/conf.py
+max-line-length = 90
```

Comment:
I'm tempted to start bumping this up on other projects too, given that is no longer the 70's

Reply:
It's kinda unacceptable for me to arbitrarily raise it in project xyz without discussing it, but I'm glad you're receptive to it in the end. I was hitting 80 chars on some simple lines with type annotations and struggling to make it not look terrible.

Reply:
I don't really mind at all. It's either easy to dial back to 80 with the assistance of tools like `black`, or it isn't easy at all in which case we'd never bother.

Reply:
(or, we just accept 90 and move on)